### ISSUE TITLE
Sidebar disappears when linking to an anchor or using the table of contents on WCAG 2.2 web page if Reduced Motion is turned on

### ISSUE THREAD

**Issue by amberhinds** [w3c/wcag#4051]
This issue pertains to https://www.w3.org/TR/WCAG22/ please let me know if I am not posting it in the right place.

In multiple browsers, after clicking one of the anchor links in the table of contents or if linking directly to the page with an anchor appended on the URL, the table of contents sidebar disappears and cannot be restored. 

Here is a [video that shows the behavior I am seeing.](https://app.screencastify.com/v3/watch/7HK2pxAi3ZWNrbhvvui5) 

There are no console errors. 

When I inspect the code, this appears to be a bug related to the prefers reduced motion CSS:

```
@media (prefers-reduced-motion: reduce) {
    body.toc-sidebar #toc {
        position: absolute;
        overflow: initial;
        bottom: unset;
    }
}
```

I do have reduced motion turned on in my OS. If I turn of reduced motion, then the sidebar stays visible when interacting with it. If I inspect the code and disable the position:absolute in the reduced motion media query above, then it works as expected.

**Comment by kfranqueiro** [comment]
Thanks for the video illustrating the issue, and sorry for the delay in response.

I've attempted to reproduce this issue (also on Firefox / Mac with Reduce motion enabled), but I'm not encountering the sidebar hiding when I follow links in the table of contents. As far as I can tell, the styles identified in the video are still present.

Are you still able to reproduce this issue in the latest version of Firefox?

**Comment by kfranqueiro** [comment]
Update: Kevin and I were both able to reproduce this issue in various Chromium-based browsers.
